ci-operator/config/kubevirt-ui/kubevirt-plugin/kubevirt-ui-kubevirt-plugin-main.yaml

```diff
@@ -0,0 +1,56 @@
+build_root:
+  project_image:
+    dockerfile_path: Dockerfile.ci
+releases:
+  initial:
+    integration:
+      name: "4.10"
+      namespace: ocp
+  latest:
+    integration:
+      include_built_images: true
+      name: "4.10"
+      namespace: ocp
+resources:
+  '*':
+    limits:
+      memory: 4Gi
+    requests:
+      cpu: 100m
+      memory: 200Mi
+tests:
+- as: kubevirt-e2e-aws
+  steps:
+    cluster_profile: aws
+    dependencies:
+      KUBEVIRT_PLUGIN_IMG_NAME: pipeline:root
```

Comment:
Abusing build root like this is an antipattern and will likely result in the job not doing what you want. Build root image is intended to serve as an image that contains various dependencies, tooling etc, not to contain the code to be tested.

Please see https://docs.ci.openshift.org/docs/architecture/ci-operator/#build-root-image, https://docs.ci.openshift.org/docs/architecture/ci-operator/#building-artifacts and https://docs.ci.openshift.org/docs/architecture/ci-operator/#building-container-images for docuemtation on how ci-operator works. If you need to produce an image to be tested, it should be built in the `images:` stanza.

Reply:
Thanks for the input @petr-muller, Will read.
I was inspired by https://github.com/openshift/release/blob/master/ci-operator/config/red-hat-storage/odf-console/red-hat-storage-odf-console-master.yaml#L36

ODF is a dynamic plugin like kubevirt-plugin, and since its merged I believed its a good example

Reply:
Yeah, that's how I noticed - someone already using this pattern (using build root image as an image to be tested) recently contacted us with issues with their setup, caused exactly by that.